Tests/test_image_access.py

```diff
@@ -168,16 +168,15 @@ def check(self, mode: str, expected_color_int: int | None = None) -> None:
             f"expected {expected_color} got {actual_color}"
         )
 
-        # Check 0
+        # check 0x0 image with None initial color
```

Comment:
Let's keep these beginning with a capital letter:

```suggestion
        # Check 0x0 image with None initial color
```

If anything, that's consistent with PEP 8: https://peps.python.org/pep-0008/#comments

> Comments should be complete sentences. The first word should be capitalized, unless it is an identifier that begins with a lower case letter (never alter the case of identifiers!).